src/python/dxpy/scripts/dx.py

```diff
@@ -4505,6 +4551,11 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
 parser_run.add_argument('--debug-on', action='append', choices=['AppError', 'AppInternalError', 'ExecutionError', 'All'],
                         help=fill("Configure the job to hold for debugging when any of the listed errors occur",
                                   width_adjustment=-24))
+parser_run.add_argument('--batch-tsv', dest='batch_tsv', metavar="FILE",
+                        help=fill('A file in tab separated value (tsv) format, with a subset ' +
+                                  'of the executable input arguments. A subjob will be launched ' +
```

Comment:
This should say a `job`, not a `subjob`, since the run hierarchy will currently be flat.